templates/customernotification.html

```diff
@@ -10,41 +10,76 @@
+                            {{ $FirstAlert = false }}
                         {{ end }}
-                        {{ if eq .Value "CephOSDCriticallyFull" }}
+                        {{ if and (eq .Value "CephOSDCriticallyFull" ) (eq $FirstAlert true ) }}
+                            Ceph OSD is Critically Full
+                            <br><br>
                             The utilization of an OSD in your cluster has exceeded 80%. Please free up some space immediately or expand the storage cluster if possible. Failure to address this issue will lead to service interruptions for the cluster.
+                            
+                            {{ $FirstAlert = false }}
                         {{ end }}
-                        {{ if eq .Value "PersistentVolumeUsageNearFull" }}
-                            The utilization of the PVC in your cluster has exceeded 75%. Please free up some space or expand the PVC if possible. Failure to address this issue will lead to service interruptions for the cluster.
+                        {{ if and (eq .Value "PersistentVolumeUsageNearFull" ) (eq $FirstAlert true ) }}
+                            Persistent Volume Usage is Nearly Full
+                            <br><br>
+                            The utilization of one of the PVs in your cluster has exceeded 75%. Please free up some space or expand the PV if possible. Failure to address this issue may lead to service interruptions for the cluster.
+                            
+                            {{ $FirstAlert = false }}
                         {{ end }}
-                        {{ if eq .Value "PersistentVolumeUsageCritical" }}
-                            The utilization of the PVC in your cluster has exceeded 85%. Please free up some space immediately or expand the PVC if possible. Failure to address this issue will lead to service interruptions for the cluster.
+                        {{ if and (eq .Value "PersistentVolumeUsageCritical" ) (eq $FirstAlert true ) }}
+                            Persistent Volume Usage Critical
+                            <br><br>
+                            The utilization of one of the PVs in your cluster has exceeded 85%. Please free up some space immediately or expand the PV if possible. Failure to address this issue may lead to service interruptions for the cluster.
+                            
+                            {{ $FirstAlert = false }}
                         {{ end }}
-                        {{ if eq .Value "CephClusterReadOnly" }}
+                        {{ if and (eq .Value "CephClusterReadOnly" ) (eq $FirstAlert true ) }}
+                            Ceph Cluster is in Read Only state
+                            <br><br>
                             Your storage cluster utilization has crossed 85% and will become read-only now! Please free up some space or if possible expand the storage cluster immediately to prevent any service access issues.
```

Comment:
```suggestion
                            Your storage cluster utilization has crossed 85%, as a result, access is now in a read-only state! Please free up some space or if possible expand the storage cluster immediately to prevent any additional service access issues.
```